.prettierignore

```diff
@@ -16,6 +16,7 @@ test
 !modules/core/src/adapter/types/shader-layout.ts
 !modules/core/src/index.ts
 !modules/engine/src/model/model.ts
+!modules/engine/src/lib/shader-factory.ts
```

Comment:
I never took the time to figure out what is happening here, why is prettier not stable for some files?

Reply:
My intention here is to tell Prettier that it *can* format this file, as an exception to all of `modules` being excluded from Prettier formatting above. Since it's a new file we might as well keep it formatted. 

However, my VSCode-Prettier plugin doesn't seem to understand these `!` negations, despite documentation saying it should work, so it's possible this line is doing nothing at all...